test/mocha/090-multinode.js

```diff
@@ -86,8 +86,9 @@ describe('Multinode', () => {
     // override elector selection to force cycling and 3f+1
     before(() => {
       let candidates;
-      const witnessSelectionApi = brLedgerNode.use('MostRecentParticipants');
-      witnessSelectionApi.api.getBlockElectors = async ({blockHeight}) => {
+      const witnessSelectionApi = brLedgerNode.use('Continuity2017');
+      witnessSelectionApi.api._witnesses.getBlockWitnesses =
```

Comment:
Do not override the core function, override the plugin function. You'll need to replace `MostRecentParticipants` with whatever the new name is for the witness pool plugin.

```suggestion
      const witnessSelectionApi = brLedgerNode.use('MostRecentParticipants');
      witnessSelectionApi.api.getBlockWitnesses =
```